core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -28,6 +28,7 @@
  * @property string  $type
  * @property string  $pageTitle
  * @property string  $language
+ * @property boolean $doNotRedirectLanguage
```

Comment:
Please also add the `find*ByDoNotRedirectLanguage()` and `countByDoNotRedirectLanguage()` methods.

Reply:
I think they're pretty stupid and unnecessary bloat, but added in c50e0ff49a880e92b3cff8caf30643ef1f572db5 🙃